burn-import/README.md

```diff
@@ -1,72 +1,13 @@
-# Burn Import: A Crate for ONNX Model Import into the Burn Framework
+# Importing Models
 
-`burn-import` facilitates the seamless import of machine learning models, particularly those in the
-ONNX format, into the Burn deep learning framework. It automatically generates Rust source code,
-aligns the model structure with Burn's native format, and converts tensor data for Burn
-compatibility.
+The Burn project supports the import of models from various frameworks, emphasizing efficiency and
+compatibility. Currently, it handles two primary model formats:
 
-> **Note**: This crate is in active development and currently supports a
-> [limited set of ONNX operators](SUPPORTED-ONNX-OPS.md).
+1. [ONNX](https://burn.dev/book/import/onnx-model.html): Facilitates direct import, ensuring the
+   model's performance and structural integrity are maintained.
```

Comment:
Always the `structural integrity` part

Reply:
Same comment as the section in the burn book.

Maybe reformulate simply to "performance and structure are maintained".